lib/lita/handlers/totems.rb

```diff
@@ -7,6 +7,24 @@ module Handlers
     class Totems < Handler
 
       @@DemoEnvironments = %w(cyan jade noir opal plum teal vert)
+      @@EnvironmentToSlackChannelMap = {
```

Comment:
not completely sure if this is the right way to hold a map. my experience is with `JS` where I would probably create a `js/json` file that holds this map and use it from there, couldn't find the "right" way to do it with Ruby

Reply:
also, this list is not complete and we are probably missing some relations (will add it later)